src/Xamarin.Android.Build.Tasks/Tasks/BundleToolBuildApkSet.cs

```diff
@@ -0,0 +1,72 @@
+﻿using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+using System.IO;
+using Xamarin.Android.Tools;
+
+namespace Xamarin.Android.Tasks
+{
+	/// <summary>
+	/// Invokes `bundletool` to create an APK set (.apks file)
+	/// 
+	/// Usage: bundletool build-apks --bundle=foo.aab --output=foo.apks
+	/// </summary>
+	public class BundleToolBuildApkSet : BundleTool
+	{
+		[Required]
+		public string AppBundle { get; set; }
+
+		[Required]
+		public string Output { get; set; }
+
+		/// <summary>
+		/// This is used to detect the attached device and generate an APK set specifically for it
+		/// </summary>
+		[Required]
+		public string AdbToolPath { get; set; }
+
+		[Required]
+		public string Aapt2ToolPath { get; set; }
+
+		[Required]
+		public string KeyStore { get; set; }
+
+		[Required]
+		public string KeyAlias { get; set; }
+
+		[Required]
+		public string KeyPass { get; set; }
+
+		[Required]
+		public string StorePass { get; set; }
+
+		public override bool Execute ()
+		{
+			//NOTE: bundletool will not overwrite
+			if (File.Exists (Output))
+				File.Delete (Output);
+
+			base.Execute ();
+
+			return !Log.HasLoggedErrors;
+		}
+
+		protected override CommandLineBuilder GetCommandLineBuilder ()
+		{
+			var adb = OS.IsWindows ? "adb.exe" : "adb";
+			var aapt2 = OS.IsWindows ? "aapt2.exe" : "aapt2";
```

Comment:
We tend to allow users to override the exe name from MSBuild e.g `Aapt2ToolExe` and them combine them in tool tasks like so
```
Path.Combine (ToolPath, string.IsNullOrEmpty (ToolExe) ? ToolName : ToolExe);
```

where `ToolName` will be the `OS.IsWindows ? "aapt2.exe" : "aapt2"`. We should probably follow the same pattern here for `aapt2` and `adb`